### PR TITLE
refactoring test base class

### PR DESCRIPTION
**Pull request by tfe2012**

### CODE REVIEW

File: Tests/Functional/TestBase.php [hunk 1]
```diff
@@ -25,11 +27,65 @@
 abstract class TestBase extends WebTestCase
```
Comment (GrandLTU) [inline Tests/Functional/TestBase.php]: Should be AbstractTestBase

Comment (saimaz) [inline Tests/Functional/TestBase.php]: `AbstractTestCase` sound more reliable

**Comment by tautrimas** [comment]
- [ ] Remove Vagrant author / commiter.

File: Tests/Functional/TestBase.php (hunk 1 above)
Comment (tautrimas) [inline Tests/Functional/TestBase.php]: It will be in another PR. Please, consult with @emaslak.
